rsocket-core/src/test/java/io/rsocket/metadata/AuthMetadataCodecTest.java

```diff
@@ -8,10 +8,10 @@
 import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-class AuthMetadataFlyweightTest {
+public class AuthMetadataCodecTest {
```

Comment:
Better not to change it right now since we have already done that migration in the master branch

Reply:
I see :) but then the Tests will need to be updated later